Automattic/wp-calypso
# Themes: Remove unused theme banners functionality

After #33213 and #33327 (cc @crunnells), theme banners state and component aren't used anywhere. It's been over a year and a half since they were used, and they were intentionally remove back then, so it should be safe to remove the remnants. That's what this PR does.

#### Changes proposed in this Pull Request

* Themes: Remove unused theme banners functionality

#### Testing instructions

* Verify the removed functionality isn't used/imported anywhere.
* Verify all tests pass.
